workspace.bzl

```diff
@@ -20,7 +20,7 @@ def _blackjack_repository_impl(ctx):
   ctx.symlink(ctx.attr._blackjack, "blackjack")
   ctx.symlink(ctx.attr.manifest, "manifest")
 
-  execution = ctx.execute(["blackjack", "manifest"])
+  execution = ctx.execute(["blackjack", "manifest", "target"])
```

Comment:
This will probably have to be something like `--target=...`.

Reply:
Could you elaborate on how exactly blackjack is called right now? I couldn't figure out how to pass down the `target` function argument to the blackjack CLI.

Reply:
Usually you'll have something like this in the `BUILD` file next to `Cargo.toml`:
```
load("@blackjack//:blackjack.bzl", "blackjack")
blackjack(name = "blackjack")
```

This will declare a `rust_binary` target `blackjack` as defined [here](https://github.com/wildarch/blackjack/blob/master/blackjack.bzl#L7). Which means really all you're doing when you run `bazel run //:blackjack` is to compile and run the rust executable that is blackjack.

If you wanted to pass a target flag to blackjack, say `--target=x86_64-apple.darwin`, you could invoke bazel like so: `bazel run //:blackjack -- --target=x86_64-apple-darwin` (note the `--` separator to make sure bazel doesn't swallow the flag itself).